server/lib/fbApi.js

```diff
@@ -444,6 +444,19 @@ const del = ({canonical = null} = {}) => {
 const wipe = () => list({fields: ['id']})
 .then(items => Promise.all(items.map(item => del({id: item.id}))));
 
+const posts = ({since} = {}) => call(
```

Comment:
This is fine - I don't think it can be optimised further. I've been unable to find a call which only returns posts of type `link`.